core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-edit-page-toolbar-seo/dot-edit-page-toolbar-seo.component.spec.ts

```diff
@@ -481,4 +481,14 @@ describe('DotEditPageToolbarSeoComponent', () => {
             });
         });
     });
+
+    describe('ngOnChange', () => {
+        it('should have a new api link', () => {
+            component.apiLink = '';
+            const expectedLink = `api/v1/page/render${componentHost.pageState.page.pageURI}?language_id=${componentHost.pageState.page.languageId}`;
+            fixtureHost.detectChanges();
+            component.ngOnChanges();
```

Comment:
Pass the responsibility to the component to trigger `onChanges`; if any input of the component changes, it should be triggered.
```
component.setInput('pageState',mockValue)
fixtureHost.detectChanges();

expect(component.apiLink).toBe(expectedLink);
```